### PR TITLE
Homework number 2

### PR DESCRIPTION
**Pull request by 40grivenprog**

### CODE REVIEW

File: HW_02/Maksim_Filipenko/maksim_filipenko_hw_02_t_01.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+puts 'The hour hand goes way in x degrees in 15 minutes'
+puts 'so...'
+x = 0
+puts 'the angle between the hours and the minutes hands is'
+puts x
```
Comment (labs-ruby) [inline HW_02/Maksim_Filipenko/maksim_filipenko_hw_02_t_01.rb]: 0 its wrong

File: HW_02/Maksim_Filipenko/maksim_filipenko_hw_02_t_02.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+puts 'Try just add 1, 3, 4, 6'
+puts 'so...'
+x = 1 / 3 + 4 * 6
```
Comment (labs-ruby) [inline HW_02/Maksim_Filipenko/maksim_filipenko_hw_02_t_02.rb]: cheat), but if you can explain why it is possible will ok!

Comment (40grivenprog) [reply HW_02/Maksim_Filipenko/maksim_filipenko_hw_02_t_02.rb]: It is possible because 1/3 = 0 but 1.0/3 = 0.3333333333333333

Comment (40grivenprog) [reply HW_02/Maksim_Filipenko/maksim_filipenko_hw_02_t_02.rb]: It is possible because 1/3 = 0 but 1.0/3 = 0.3333333333333333

**Pull request merged**